src/controllers/main/main.ts

```diff
@@ -1522,11 +1530,14 @@ export class MainController extends EventEmitter {
     this.emitUpdate()
   }
 
+  // TODO: Figure out better name for this method
+  // TODO: Re-use `withStatus` instead of the broadcastStatus flag
+  // TODO: The `broadcastStatus` flag gets re-used here and for signing an
+  // account op too, that could lead to troubles. It should be separated.
   async broadcastSignedMessage(signedMessage: SignedMessage) {
```

Comment:
I suggest removing that method and large comment and integrating the code directly into the signMessageSign function because we really do not broadcast anything